README.md

```diff
@@ -1,2 +1,97 @@
 # react-ga-onetrust-consent
-React implementation of OneTrust consent banner integrated with Google Analytics signals.
+
+React implementation of OneTrust consent banner integrated with Google Analytics signals. Initially built to support analytics-enabled University of Michigan Teaching & Learning applications, this logic using Google Analytics tracking alongside OneTrust consent verification is written to be properly standardized; We will aim to iterate on this project such that it could be shared with other UofM web apps that utilize React and Google Analytics. 
+
+For more information on the University of Michigan's technical implementation of OneTrust, see the OVPC Digital team's [instructions on cookie disclosures](https://vpcomm.umich.edu/resources/cookie-disclosure/).
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Development](#development)
+- [Testing](#testing)
+- [Releasing](#releasing)
+- [License](#license)
+
+# Overview
+
+
+Applications should import and apply the hook `useGoogleAnalytics` at the root level of App in the DOM. Upon initializing with the proper settings, this hook will call upon the `useOneTrust` hook which initializes the consent banner and captures user preferences.
+
+Beyond simply loading a script for the banner, the `useOneTrust` hook also writes a callback to properly emit settings changes to Google Analytics and delete Google Analytics cookies if tracking is denied (EU countries only). To get a better understanding how OneTrust and Google Analytics work together, see the [UofM Google Analytics implementation instructions](https://vpcomm.umich.edu/resources/cookie-disclosure/#3rd-party-google-analytics).
+
+Remember to keep in mind that the consetn banner h
+
+
+# Installation
+You can install this package via GitHub Packages. Add the following configuration to your .npmrc file to use GitHub Packages registry:
```

Comment:
where is this file `.npmrc` located? on local machine? where at the project level or Home directory ( for Macs)

Reply:
After discovering the alternative discussed in the dev meeting, I added a disclaimer in the readme that npmrc is no longer necessary.

Reply:
Nice